src/models/Control.php

```diff
@@ -119,24 +119,24 @@ class Control extends Model {
   }
 
   public static async function importGame(): Awaitable<bool> {
-    $data_game = self::readJSON('game_file');
-    if ($data_game) {
-      $logos = must_have_idx($data_game, 'logos');
+    $data_game = JSONImporterController::readJSON('game_file');
+    if ($data_game instanceof KeyedContainer) {
+      $logos = array_pop(must_have_idx($data_game, 'logos'));
       $logos_result = await Logo::importAll($logos);
```

Comment:
Does this pass typechecking (`hh`)? The documentation says `array_pop` can return null, but `Logo::importAll` can't take null.